Destination/Destination.php

```diff
@@ -3,15 +3,25 @@
 namespace Zenstruck\BackupBundle\Destination;
 
 use Psr\Log\LoggerInterface;
+use \Countable;
+use \IteratorAggregate;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-interface Destination
+interface Destination extends Countable, IteratorAggregate
```

Comment:
It is matter of coding style, destination holds collection of backups, so it is aggregating object.

However, it is your library, we can replace that with method 'getBackups' and achieve same result.

I am for any approach, booth are correct and I do not mind to follow coding style of owner - if you do not like something, do yell :-) no problem.

N.

Sent from my iPad

> On Sep 8, 2015, at 5:12 PM, Kevin Bond <notifications@github.com> wrote:
> 
> In Destination/Destination.php:
> 
> >  
> >  /**
> >   * @author Kevin Bond <kevinbond@gmail.com>
> >   */
> > -interface Destination
> > +interface Destination extends Countable, IteratorAggregate
> Having the destination interface Countable/IteratorAggregate seems strange to me. It makes it seem like there are multiple desinations when in fact, there are multiple Backups.
> 
> —
> Reply to this email directly or view it on GitHub.
>